snapcraft/internal/errors.py

```diff
@@ -301,7 +340,10 @@ def __init__(self, plugin_name, properties):
 
 class StagePackageDownloadError(SnapcraftError):
 
-    fmt = 'Error downloading stage packages for part {part_name!r}: {message}'
+    fmt = (
+        'Failed to fetch stage packages: '
+        'Error downloading stage packages for part {part_name!r}: {message}.'
```

Comment:
This addition doesn't add meaning, it seems redundant. Perhaps this refactor is good enough?

```
'Failed to fetch stage packages for {part_name!r}: '
'{message}.'
```

Reply:
I'm trying to stick with what+why here.
What: failed to fetch stage packages. Why: they couldn't be downloaded.
Another reason to fail to fetch could be that the cache storage is broken, or something maybe?